lib/Demangling/OldRemangler.cpp

```diff
@@ -1226,14 +1226,17 @@ void Remangler::mangleBuiltinTypeName(Node *node) {
   } else if (stripPrefix(text, "Builtin.Float")) {
     Out << 'f' << text << '_';
   } else if (stripPrefix(text, "Builtin.Vec")) {
-    auto split = text.split('x');
-    Out << 'v' << split.first << 'B';
-    if (split.second == "RawPointer") {
+    // Avoid using StringRef::split because its definition is not
+    // provided in the header so that it requires linking with libSupport.a.
+    size_t splitIdx = text.find('x');
+    Out << 'v' << text.substr(0, splitIdx) << 'B';
+    auto Element = text.substr(splitIdx).substr(1);
```

Comment:
Other places in the diff you are using camelCase. Can you use that here for `Element` as well?

Reply:
OK. I'm always torn between trying to match the LLVM naming convention vs. the existing code in a file. Better to match one or the other, I guess.